public/index.js

```diff
@@ -42,3 +45,5 @@ map.addControl(new maplibregl.NavigationControl({ showCompass: false }));
 map.addControl(new maplibregl.ScaleControl());
 
 map.addControl(new maplibregl.GeolocateControl());
+
+map.addControl(createGeocoderControl());
```

Comment:
das hier schiebt die suche nach oben links in die Karte.
Find ich fürs erste besser als die aktuelle Position. 

```suggestion
map.addControl(createGeocoderControl(), 'top-left');
```

Reply:
in deinem code machst du das einbinden etwas zu äufwändig. so wie hier sollte es einfacher und direkter gehen:

> bzw. man kann das Control auch in den Header schieben auf diese Weise:
> 
> 
> 
> ```js
> 
> [...]
> 
> const geocoderControl = createGeocoderControl();
> 
> geocoderControl.addTo('#geocoder-container');
> 
> [...]
> 
> ```
> 
> 
> 
> HTML:
> 
> 
> 
> ```html
> 
> [...]
> 
>     <header class="header">
> 
>       <a href="/">
> 
>         <img src="/img/osmde.svg" alt="OpenStreetMap Deutschland" class="logo">
> 
>       </a>
> 
>       
> 
>       <div id='geocoder-container'></div>
> 
> 
> 
> [...]
> 
> ```
> 
> 
> 
>